packages/mrm/src/index.js

```diff
@@ -141,16 +142,42 @@ function runTask(taskName, directories, options, argv) {
 				new MrmInvalidTask(`Cannot call task “${taskName}”.`, { taskName })
 			);
 		}
+		const configPromise = argv.interactive
+			? getInteractiveConfig(module, options)
+			: options;
 
-		console.log(kleur.cyan(`Running ${taskName}...`));
-		Promise.resolve(module(getConfigGetter(options), argv))
-			.then(() => {
-				resolve();
+		Promise.resolve(configPromise)
+			.then(getConfigGetter)
+			.then(config => {
+				console.log(kleur.cyan(`Running ${taskName}...`));
+				return module(config, argv);
 			})
-			.catch(reject);
+			.catch(reject)
+			.then(resolve);
 	});
 }
 
+/**
+ * Get task specific interactive config options.
+ *
+ * @param {Object} task
+ */
+function getInteractiveConfig(task, initials) {
```

Comment:
How do we define default values?

Reply:
There are multiple sources for "defaults" here, which follow this order:

1. Task declared defaults;
2. Global config files (i.e. `~/.mrm/config.json`);
3. Command line arguments (i.e. `--config:foo`).

All these are addressed in this PR.